### DISCUSSION TITLE
GitHub reorganisation

### DISCUSSION THREAD

**Discussion by sndrs** [guardian/csnx#440] [Team Posts]
So I think teams are not really intended to be used this way. Really we created a team to be able to use discussions (which seem to have worked really well?). 

But I think teams are _intended_ to be something not everyone is in by default. So, for instance, a team could be codeowner on a repo, or pingable in a comment.

Since we only really created this team for discussions, I'm thinking this might be a good move:

- public focus and discussion moves to https://github.com/guardian/client-side-infra
  - I've added discussions now they're in public beta
  - means anyone with a GitHub account can take part
- move the various packages in https://github.com/guardian/configs and `@guardian/libs` packages into https://github.com/guardian/client-side-infra
  - make it a monorepo that dogfoods itself
  - set up semantic release/CD etc on the packages in it
  - client-side-infra team would be codeowner/admin of the repo
- reduce members of client-side-infra team to anyone who wants the responsibility of maintaining that repo and those packages, and be pingable in other GitHub places etc
  - a review from codeowners would be required on PRs to packages in client-side-infra, so it would involve a bit of work

what do people think?

**Comment by tjmw** [comment]
This sounds really great, thanks for investigating Alex!

> move the various packages in https://github.com/guardian/configs and @guardian/libs packages into https://github.com/guardian/client-side-infra

I like this idea, it seems like it'll have the added benefit of increasing discoverability of these packages too.

Do you think it's possible to migrate existing discussions from this group to the new repo? There's loads of interesting stuff in there so would be great to keep it.

**Comment by sndrs** [comment]
> Do you think it's possible to migrate existing discussions from this group to the new repo? There's loads of interesting stuff in there so would be great to keep it.

not that I can find, which I agree is a big shame. we can try to make them as discoverable as possible by linking between them though. I def think they should be preserved

**Comment by sndrs** [comment]
> it seems like it'll have the added benefit of increasing discoverability of these packages too

yeah I thought this too, I hope it will. you can imagine a nice readme explaining them all etc. also means just one repo to look after (albeit with added complexity)

**Comment by sndrs** [comment]
update on this – turns out we don't need to move the discussions, you only need to be a member of the guardian org, not this team to read/contribute/create a discussion. so they can all stay as they are 👯
